jabgui/src/main/java/org/jabref/gui/WelcomeTab.java

```diff
@@ -124,7 +131,28 @@ private VBox createWelcomeStartBox() {
                 stateManager, fileUpdateMonitor, entryTypesManager, undoManager, clipBoardManager,
                 taskExecutor).execute());
 
-        return createVBoxContainer(startLabel, newLibraryLink, openLibraryLink);
+        Hyperlink openExampleLibraryLink = new Hyperlink(Localization.lang("Open example library"));
+        openExampleLibraryLink.getStyleClass().add("welcome-hyperlink");
+        openExampleLibraryLink.setOnAction(e -> {
+            try (InputStream in = getClass().getResourceAsStream("/Chocolate.bib")) {
+                if (in != null) {
+                    Path tmpFile = Files.createTempFile("example-library", ".bib");
+                    try {
+                        Files.copy(in, tmpFile, StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Wrong approach.

1. Create a new library
2. Fill it with contents of `Chocolate.bib`. For that, you need to use `org.jabref.logic.importer.fileformat.BibtexParser#BibtexParser(org.jabref.logic.importer.ImportFormatPreferences)`. Then you can call org.jabref.model.database.BibDatabase#insertEntries(java.util.List<org.jabref.model.entry.BibEntry>) - similar to `addExampleButton.setOnAction(...`